trade-finance-manager-ui/server/services/utilisation-report-service.js

```diff
@@ -0,0 +1,49 @@
+const { format, subMonths } = require('date-fns');
+const api = require('../api');
+const { getBusinessDayOfMonth } = require('../helpers/date');
+const { BANK_HOLIDAY_REGION } = require('../constants');
+
+/**
+ * Fetches a list of bank holiday dates for the specified UK region
+ * @param {string} userToken - Token to validate session
+ * @param {'england-and-wales' | 'scotland' | 'northern-ireland'} region
+ * @returns {Promise<Date[]>}
+ */
+const getBankHolidays = async (userToken, region) => {
+  const bankHolidays = await api.getUkBankHolidays(userToken);
+  return bankHolidays[region].events.map((event) => new Date(event.date));
+};
```

Comment:
Could we maybe make it so that `region` is instead an array of `regions` so that, if you wanted, you could get the holidays for england and wales and northern island in the same call of this function?

Reply:
Not a bad idea, but we don't need it currently so it doesn't feel worth spending the time implementing it. If we do need it in the future we can just implement it then.